cmd/operator/main.go

```diff
@@ -448,7 +448,7 @@ func Main() int {
 	srv := &http.Server{
 		Handler:   mux,
 		TLSConfig: tlsConfig,
-		ErrorLog:  stdlog.New(log.NewStdlibAdapter(logger), "", stdlog.LstdFlags),
+		ErrorLog:  stdlog.New(log.NewStdlibAdapter(logger), "", 0),
```

Comment:
could you add a comment explaining why we set the flag to 0?

Reply:
Added comment and further improved generated log entry with better flag setting.

We now have an appropriate and consistent `caller` field:
```
{"caller":"server.go:3093","msg":"http: TLS handshake error from 127.0.0.1:59116: EOF","ts":"2021-04-10T19:17:06.7564989Z"}
```
Instead of begin mistakenly extracted (parsed) from the message as before:
```
{"caller":"http: TLS handshake error from 127.0.0.1:36752","msg":"EOF","ts":"2021-04-09T01:55:08.2770325Z"}
```